assistant_dists/dream_kg/docker-compose.override.yml

```diff
@@ -238,9 +239,9 @@ services:
     deploy:
       resources:
         limits:
-          memory: 128M
+          memory: 512M
```

Comment:
did you increase the value in component/container cards?

Reply:
No, this should stay unchanged.
Reverted